frontend/public/babel.config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function(api) {
+    const isProduction = api.env("production");
+    const isTest = api.env("test");
+    return {
+        presets: isTest ? [
+            "@babel/env",
+            "@babel/preset-react"
+        ] : [
+          ["@babel/preset-env", { modules: false }],
+          "@babel/preset-react",
+          "@babel/preset-flow"
+        ],
+        plugins: [
+          "@babel/plugin-transform-flow-strip-types",
+          "react-hot-loader/babel",
```

Comment:
This is getting used in all environments right now. Any harm in that, or should it be non-prod only?